testsuite/MDAnalysisTests/coordinates/test_fhiaims.py

```diff
@@ -0,0 +1,208 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import absolute_import
+
+import pytest
+from six import StringIO
+from six.moves import zip
+import os
+
+import MDAnalysis as mda
+import numpy as np
+from MDAnalysisTests import make_Universe
+from MDAnalysisTests.coordinates.base import (
+    _SingleFrameReader, BaseWriterTest)
+from MDAnalysisTests.datafiles import FHIAIMS
+from numpy.testing import (assert_equal,
+                           assert_array_almost_equal,
+                           assert_almost_equal)
+
+from six import StringIO
+
+@pytest.fixture(scope='module')
+def universe():
+    return mda.Universe(FHIAIMS, FHIAIMS)
+
+@pytest.fixture(scope='module')
+def universe_from_one_file():
+    return mda.Universe(FHIAIMS)
+
+@pytest.fixture(scope='module')
+def ref():
+    return RefFHIAIMS()
+
+class RefFHIAIMS(object):
+    from MDAnalysis.coordinates.FHIAIMS import (FHIAIMSReader, FHIAIMSWriter)
+
+    filename, trajectory, topology = [FHIAIMS] * 3
+    reader, writer = FHIAIMSReader, FHIAIMSWriter
+    pos_atom1 = np.asarray(
+        [6.861735,   2.103823,  37.753513], dtype=np.float32)
+    dimensions = np.asarray(
+        [18.6, 18.6,  55.8, 90., 90., 90.], dtype=np.float32)
+    n_atoms = 6
+    n_frames = 1
+    time = 0.0
+    ext = '.in'
+    prec = 6
+    container_format = True
+    changing_dimensions = False
+
+
+class TestFHIAIMSReader(object):
```

Comment:
Is there a test for testing the`atom_frac` reading (for all and some)?

Reply:
there now